### PR TITLE
Added Scheduler for messages

### PR DESCRIPTION
**Pull request by Rits1272**
A user now will be able to define a specified time to deliver a particular message. The message will then be sent at the specified time.

How to use:

**python3 -m wplay -sch "username"** OR **python3 -m wplay --schedule "username"**

The user then has to input the specified time in 24 hour format **(HH:MM:SS)**. The message then get delivered at the specified time.

### CODE REVIEW

**Comment by Rits1272** [comment]
Closes #209. Please review @rpotter12

File: wplay/scheduleMessage.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    delta_hour = int(hour) - current_time.hour
+    delta_min = int(minute) - current_time.minute
+    delta_second = int(second) - current_time.second
+    total_seconds = delta_hour*3600 + delta_min*60 + delta_second
+    if total_seconds < 0:
+        print("Current time is ahead of the scheduled time")
+        sys.exit()
+    message = io.ask_user_for_message_breakline_mode()
+    print("Your message is scheduled at : ", time_)
+
```
Comment (rpotter12) [inline wplay/scheduleMessage.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5198274)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Complexity increasing per file
==============================
- wplay/scheduleMessage.py  3
- wplay/__main__.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5198274&bid=17048508)

File: wplay/__main__.py [hunk 2]
```diff
@@ -102,6 +109,9 @@ def get_arg_parser():
         except (IndexError, ValueError):
             bID = 0
         savechat.runMain('pull', str(args.target), bID)
+
```
Comment (rpotter12) [inline wplay/__main__.py]: Codacy Issue found: [Trailing whitespace](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5198274)

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- wplay/scheduleMessage.py  3
- wplay/__main__.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5198274&bid=17048508)

**Comment by rpotter12** [comment]
@Rits1272 Please add your work's execution in the PR comment :)

**Comment by Rits1272** [comment]
Please check the description @rpotter12 :)

**Comment by Rits1272** [comment]
Can I create an issue for adding the information regarding this in the **readme** file? @rpotter12 @xandao6

**Comment by rpotter12** [comment]
@Rits1272 Please resolve the conflicts.

**Comment by Rits1272** [comment]
Resolved the conflicts. Please check @rpotter12!

**Comment by rpotter12** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- wplay/scheduleMessage.py  3
- wplay/__main__.py  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/rpotter12/whatsapp-play/pullRequest?prid=5198274&bid=17048508)

**Pull request merged**